src/versioning/Versioning.ts

```diff
@@ -18,7 +19,14 @@ export default class Versioning {
    * Return string representation of SDK version
    */
   static get sdkVersion(): string {
-    return '1.19.14';
+    return semverString;
+  }
+
+  /**
+   * Return the SHA-1 of the Git commit from which this build was created.
+   */
+  static get buildSHA(): string {
```

Comment:
I assume this will fail the unit test coverage :D

Reply:
Hah, I believe you're right!